classes/ActionScheduler_ListTable.php

```diff
@@ -0,0 +1,354 @@
+<?php
+
+/**
+ * Implements the admin view of the actions.
+ */
+class ActionScheduler_ListTable extends PP_List_Table {
+	/**
+	 * The package name. It is used also as the domain for the translations
+	 */
+	protected $package = 'action-scheduler';
+
+	/**
+	 * Columns to show (name => label). The label is automatically
+	 * translated before rendering
+	 */
+	protected $columns = array(
+		'hook'   => 'Hook',
+		'status' => 'Status',
+		'args'   => 'Arguments',
+		'group'  => 'Group',
+		'recurrence' => 'Recurrence',
+		'scheduled'  => 'Scheduled Date',
+		'claim_id'   => 'Claim ID',
+		'comments'   => 'Log',
+	);
+
+	protected $row_actions = array(
+		'hook' => array(
+			'run' => array( 'Run', 'Process the action now as if it were run as part of a queue' ),
+		),
+	);
+
+	/**
+	 *  The active data stores
+	 */
+	protected $store;
+
+	/**
+	 * Bulk actions. The key of the array is the method name of the implementation:
+	 *
+	 *     bulk_<key>(array $ids, string $sql_in).
+	 *
+	 * See the comments in the parent class for further details
+	 */
+	protected $bulk_actions = array(
+		'delete' => 'Delete',
+	);
+
+
+	/**
+	 * If it is true it will load our own javascript library.
+	 *
+	 * Our javascript library will show the logs as a jQuery modal.
+	 */
+	protected static $should_include_js = false;
+
+
+	/**
+	 * Sets the current data store object into `store->action` and initialises the object.
+	 */
+	public function __construct() {
+		self::$should_include_js = true;
+
+		$this->store = (object) array(
+			'action' => ActionScheduler_Store::instance(),
+			'log'    => ActionScheduler_Logger::instance(),
+		);
+
+		parent::__construct( array(
+			'singular' => $this->translate( 'action-scheduler' ),
+			'plural'   => $this->translate( 'action-scheduler' ),
+			'ajax'     => false,
+		) );
+	}
+
+	/**
+	 * Setups the libraries. It is executed if `is_admin()` is TRUE.
```

Comment:
Instead of `Setups`, the correct English would be `Sets up`. Also this isn't a library, it's a class. :)